src/Essential.Diagnostics.SeqTraceListener/Diagnostics/SeqBatchSender.cs

```diff
@@ -38,7 +38,17 @@ public void Enqueue(TraceData traceData)
             }
             else
             {
-                PostBatch(new[] { traceData });
+                try
+                {
+                    PostBatch(new[] { traceData });
+                }
+                catch
+                {
+                    if (!_associatedTraceListener.SyncErrorHandling)
+                    {
+                        throw;
+                    }
```

Comment:
If ignoring (suppressing), i.e. the else of the above if, then consider writing to Console.Error.WriteLine (with a guard on Console.Error), to output that the send failed but the error is being ignored.

Reply:
Would you like it implemented? if so could you explain what you mean with Guard?

Reply:
Have a look at the async batch processing where it retries; there is no internal log so it writes to Console.Error, but first of all checks it has a value, i.e. if (Console.Error != null) then { Console.Error.WriteLine(....); }

Although checking the documentation I'm not sure that is possible (e.g. SetError() throws an exception if a null is passed in). Maybe I was just being overly cautious.

Reply:
Something like this? Or are you looking for something different?

Reply:
@sgryphon Can you have a look if you want something to bechanged still? Or is it the way you would like it to be.